demisto_sdk/commands/common/errors.py

```diff
@@ -224,6 +224,7 @@
     "changed_integration_yml_fields": {'code': "IN147", "ui_applicable": False, 'related_field': 'script'},
     "parameter_is_malformed": {'code': "IN148", 'ui_applicable': False, 'related_field': 'configuration'},
     'empty_outputs_common_paths': {'code': 'IN149', 'ui_applicable': False, 'related_field': 'contextOutput'},
+    "empty_command_arguments": {'code': 'IN150', 'ui_applicable': False, 'related_field': 'arguments'},
```

Comment:
Not sure whether the `ui_applicable` field should be true or false. Let me know what you think